### ISSUE TITLE
New Update Crashes with this mod

### ISSUE THREAD

**Issue by GaIaxyGamer** [rolopogo/CustomPlatforms#5]
After the hotfix launching the game with this mod will bring you to the health and safety disclaimer and pressing ok will work as expected however after it loads SteamVR because Beat Saber crashes

**Comment by rolopogo** [comment]
I've just published version 2.5.0 which addresses some stability problems.
Also check that you have CustomUI installed https://github.com/brian91292/BeatSaber-CustomUI or you are likely to crash.

**Comment by rolopogo** [comment]
I've found what I believe to be the issue:
SongLoader has change the way they manage scenes, which has caused conflicts with the Scene Override system from platforms.
I have submitted a pull request to SongLoader to fix the conflict.

For now, to get your game working again with Platforms installed, navigate to BeatSaber/UserData/modprefs.ini. Open with a text editor and find the CustomPlatforms region. Change the line:
`EnvironmentOverrideMode=3` to `EnvironmentOverrideMode=0`

Until Songloader is patched, make sure you do not change the environment override away from "None" in the in settings menu.
